src/init/init.ts

```diff
@@ -124,11 +124,7 @@ function createYeomanEnvironment(): Promise<any> {
     env.registerStub(ApplicationGenerator, 'polymer-init-application:app');
     env.registerStub(shopGenerator, 'polymer-init-shop:app');
     env.registerStub(pskGenerator, 'polymer-init-starter-kit:app');
-    env.lookup((error) => {
-      if (error) {
-        reject(error);
-        return;
-      }
+    env.lookup(() => {
```

Comment:
Is it really ok to remove the error argument? An error will cause the Promise to never settle. The Yeoman docs say the callback takes an error.